### PR TITLE
Build: Remove squashfs-root folder after appimage build

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Fixes part of #1959 .

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

## Explain Changes

Are the commit messages enough? If not, elaborate.

### CODE REVIEW

**Review by mssalvatore — CHANGES_REQUESTED** [review]
Both the `docker.sh` and `appimage.sh` scripts now have cleanup actions that need to be taken. Add a `cleanup()` function to the docker script and have `build_package.sh` call it.

File: build_scripts/docker/docker.sh [hunk 1]
```diff
@@ -78,9 +70,9 @@ move_package_to_dist_dir() {
     mv "$1" "$2/"
 }
 
-clean_stale_images() {
-   echo "Cleaning images"
+cleanup() {
+   echo "Cleaning docker images"
 
-   sudo docker rmi $1
+   sudo docker rmi "$DOCKER_IMAGE_NAME:$1"
```
Comment (mssalvatore) [inline build_scripts/docker/docker.sh:76]:
```python
cleanup() {
   tag = $1
   echo "Cleaning docker images"

   sudo docker rmi "$DOCKER_IMAGE_NAME:$tag"
```

File: build_scripts/build_package.sh [hunk 2]
```diff
@@ -209,7 +209,15 @@ fi
 
 setup_build_dir "$agent_binary_dir" "$monkey_repo" "$deployment_type" "$is_release_build"
 
-build_package "$monkey_version" "$commit_id" "$DIST_DIR"
+if [ -n "$monkey_version" ]; then
+    monkey_version="v$monkey_version"
+else:
+    monkey_version="$commit_id"
+fi
```
Comment (mssalvatore) [inline build_scripts/build_package.sh:216]: Maybe wrap in a function that returns the new value?

```bash
format_version() {
  unformatted_version = $1

  if [ -n "$unformatted_version ]; then
    echo "v$unformatted_version"
...
}

File: build_scripts/docker/docker.sh [hunk 3]
```diff
@@ -37,17 +38,10 @@ copy_server_config_to_build_dir() {
...
   pushd ./docker
 
-  if [ -n "$1" ]; then
-    version="v$version"
-  else
-    version="$commit_id"
-  fi
-
-  docker_image_name="guardicore/monkey-island:$version"
+  docker_image_name="$DOCKER_IMAGE_NAME:$version"
```
Comment (mssalvatore) [inline build_scripts/docker/docker.sh:44]: I don't love having `DOCKER_IMAGE_NAME` and `docker_image_name`. I can't find a term for a name and a tag together, so I'm not really sure what else to call it.

What about removing this variable and just building the string inline? It only gets used in one place.

`build_docker_image_tar "$DOCKER_IMAGE_NAME:$version" "$tar_name"`

Comment (ilija-lazoroski) [reply build_scripts/docker/docker.sh:44]: I didn't like it either but I left it for readability. I will do it inline.

**Review by mssalvatore — APPROVED** [review]
Approved as long as `shellcheck` has no complaints.

**Pull request merged**